generation/anomaly_detection/2d_classifierfree_guidance_anomalydetection_tutorial.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<p>Like <a href="https://github.com/Project-MONAI/tutorials/blob/main/generation/2d_ldm/2d_ldm_tutorial.ipynb" target="_blank">here</a> the next cell down needs to be the "Setup environment" cell followed by imports. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>